docs/ARCHITECTURE.md

```diff
@@ -24,13 +24,18 @@ Operator Capability Manager (OCM) implements several checks on TensorFlow operat
 
 Graph partitioner examines the operators that are marked for clustering by OCM and performs further analysis on them. In this stage, the marked operators are first assigned to clusters. Some clusters are dropped after the analysis. For example, if the cluster size is very small or if the cluster is not supported by the backend after receiving more context, then the clusters are dropped and the operators fall back on native TensorFlow runtime. Each cluster of operators is then encapsulated into a custom operator that is executed in OpenVINO™ runtime.
 
+#### TensorFlow Frontend
+
+[TensorFlow Frontend](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow) converts the TensorFlow operations in the clusters to OpenVINO™ Model
```

Comment:
Hi, I have a few suggestions.

Here we talk about what TFFE does. But, can we add a bit more detail on what TFFE is and what its features and advantages are? For instance, we can write that it's an OpenVINO component replacing TF Importer in OVTF now for graphs translations because they are now maintained better and likely to get more efficient in the future?

It is also worth considering if we can make a list of supported Ops in TFFE. Maybe for now we can forward to the **op** folder? https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow/src/op

Reply:
I believe it might be tricky to explain why a translator is better than the other one (We can make both translators identical if we want). Certainly it has advantages as we know but for the OVTF architecture section, we already describe the internal components and where TF FE and TF Importer.
For the improvements over 2.1.0 and the impact of TF FE, I think the release notes might be a better place to add these changes.